src/main/java/org/folio/dcb/service/impl/PatronGroupServiceImpl.java

```diff
@@ -19,12 +19,16 @@ public class PatronGroupServiceImpl implements PatronGroupService {
   public String fetchPatronGroupIdByName(String groupName) {
     log.debug("fetchPatronGroupIdByName:: Fetching patron group details with groupName {} ",
       groupName);
-    return groupClient.fetchGroupByName("group==" + groupName)
+    return groupClient.fetchGroupByName(queryByGroupName(groupName))
       .getUsergroups()
       .stream()
       .filter(group -> group.getGroup().equals(groupName))
       .findFirst()
       .map(UserGroup::getId)
       .orElseThrow(() -> new NotFoundException(String.format("Patron group not found with name %s ", groupName)));
   }
+
+  private String queryByGroupName(String groupName) {
```

Comment:
Usually, methods start with a verb, in this case `query` is a verb, but the method does not query any data, 
so
```
getQueryByGroupName(...)
``